src/watts/data.py

```diff
@@ -0,0 +1,190 @@
+# SPDX-FileCopyrightText: 2011-2023 Massachusetts Institute of Technology, UChicago Argonne LLC, and OpenMC contributors
```

Comment:
`data.py` is not very explicit name - since this is for MCNP, I recommend naming this `data_mcnp.py` or something even more explicit.

Reply:
I'm open to other names, but the data in here is not at all specific to MCNP (atomic symbols, natural abundances) so `data_mcnp.py` wouldn't make sense

Reply:
maybe `data_elements.py` or `data_isotope.py` ?
I am thinking we will have other `data.py` files in the future and will need to differentiate them.

Reply:
I ended up renaming to `fundamental_data.py`